src/index.scss

```diff
@@ -106,6 +106,18 @@ h3 {
     background-color: $standard-red;
     border-color: $standard-red;
   }
+
+  &--red-outlined {
```

Comment:
nit: i much prefer BEM classes over modifier classes like this, but we have no established style-guide yet so this is fine

Reply:
This is my first time learning of BEM...interesting :)